crates/consensus/src/transaction/envelope.rs

```diff
@@ -408,6 +414,12 @@ impl<Eip4844: RlpEcdsaEncodableTx> EthereumTxEnvelope<Eip4844> {
         matches!(self, Self::Eip7702(_))
     }
 
+    /// Returns true if the transaction supports dynamic fees.
+    #[inline]
+    pub const fn is_dynamic_fee(&self) -> bool {
+        matches!(self, Self::Legacy(_) | Self::Eip2930(_) | Self::Eip1559(_) | Self::Eip4844(_) | Self::Eip7702(_))
+    }
```

Comment:
please remove this

Reply:
Have made changes, do check them out once you're free!